index.js

```diff
@@ -55,11 +55,10 @@ export default class Dots extends Component {
     activeBorderColor: "#FFF"
   };
 
-  componentWillReceiveProps({ active }) {
-    const oldActiveIndex = this.props.active;
-
-    if (oldActiveIndex !== active) {
-      this.scrollTo(active);
+  componentDidUpdate (prevProps, prevState, snapshot) {
```

Comment:
The **prevState** and **snapshot** parameters are never used.

Reply:
My apologies, my IDE added these automatically when auto-completing the function. I have updated this to remove them.